### PR TITLE
completed week 1 assessment

### PR DESCRIPTION
**Pull request by JacksonPruett**
# Submitting Week 1 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [X] Coding Challenges
- [X] Coding Concepts
- [X] Interview Questions
- [X] Career Readiness

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

I wasn't sure what sort of format my responses in the interview questions and pseudo code were supposed to be. I remember sorta the way Will did it during the instructor examples but thought maybe that was more for demonstration purposes to show what information goes where. If that's how it should be typed every time then let me know.

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -26,20 +26,36 @@ const fruit2 = "banana"
...
 // Expected output: 9
 
 // Pseudo code:
+// Adding the length of my first and second arrays to get my result.
 
 // --------------------2) Create the code that will reverse the letters of a string using the test variable provided below.
 
-const currentCohort = "Golf 2023"
-// Expected output: "3202 floG"
+const currentCohort = "Hotel NOT Golf 2023"
```
Comment (chrisyhlee) [inline code-challenges.js:40]: Haha

File: code-challenges.js [hunk 2]
```diff
@@ -26,20 +26,36 @@ const fruit2 = "banana"
...
+for (let i = 0; i < stockExchange.length; i++){
+    if (stockExchange[i] % 2 !== 0){
+        console.log(stockExchange[i])
+    }      
+}
+
 // Expected output: 13 5 -5 27
 
 // Pseudo code:
+// Honestly I don't really know how to explain it. The for loop tells the computer to perform a cycle at index 0 of the array, that index 0 is less than the length of the array and to go the length of the array in increments of 1. If the variable index divided by two does not equal 0 then print out that number.
```
Comment (chrisyhlee) [inline code-challenges.js:61]: Here's an example of good pseudo code for this code:
- start iteration using a for loop
- each iteration checks if the current value of the iteration index in the array does not have a remainder of 0 when divided by 2
- if that's the case, the value gets logged in the console, and the iteration continues

You can use this same format for all pseudo code

**Review by chrisyhlee — CHANGES_REQUESTED** [review]
To answer your question, you can use the same format that Will showed for your own pseudo code and interview questions. Remember, researched answers in the interview questions should not be copy and pasted. This assessment currently displays Novice. I'm seeing some confusion on some of the concepts in the code, and also I see that explaining the code seems to be challenging for you right now. That's okay, it will improve with repetition and practice, so keep up the hard work! In order to get this pull request approved, you will need to add your "about me" statement and your linked in URL to the career-readiness.md file. Once you add those, you can add, commit, and push to the same branch, and your pull request will update automatically.

**Comment by JacksonPruett** [comment]
Understood. I’ll nail the next assessment.

**Review by chrisyhlee — APPROVED** [review]

**Pull request merged**